web/public/mgmt/config.yml

```diff
@@ -14,6 +14,11 @@ local_backend: true
 media_folder: web/public/img
 logo_url: https://dev.navigator.business.nj.gov/img/Navigator-logo.svg
 public_folder: web/public
+slug:
+  encoding: "ascii"
+  clean_accents: true
+  sanitize_replacement: "_"
```

Comment:
What does this do?

Reply:
Stops people from putting weird characters in the slug field, like Unicode stuff, but I can remove that from here